client/src/document/baseline-indicator.tsx

```diff
@@ -168,7 +171,7 @@ export function BaselineIndicator({ status }: { status: SupportStatus }) {
               target="_blank"
               rel="noreferrer"
             >
-              <span className="visually-hidden">Feedback</span>
+              Give feedback
```

Comment:
In AI Help 2.0 this will be called "Report Feedback":

```suggestion
              Report Feedback
```

Reply:
I'll update the copy but keep the capitalization (so it's consistent with the other links in the widget):

```suggestion
              Report feedback
```